server/service/http/api.go

```diff
@@ -178,3 +179,27 @@ func (a *API) route(writer http.ResponseWriter, req *http.Request) {
 
 	a.respond(writer, string(resultByte))
 }
+
+type DeleteTableRequest struct {
+	ClusterName string `json:"clusterName"`
+	SchemaName  string `json:"schemaName"`
+	Table       string `json:"table"`
+}
+
+func (a *API) deleteTable(writer http.ResponseWriter, req *http.Request) {
+	var deleteTableRequest DeleteTableRequest
+	err := json.NewDecoder(req.Body).Decode(&deleteTableRequest)
+	if err != nil {
+		log.Error("decode request body failed", zap.Error(err))
```

Comment:
Print `req.body` in log info.

Reply:
Request body has been print in `printRequestInsmt`, it will print every request information and details, so it is not needed in here.